layouts/partials/jumbotron.html

```diff
@@ -0,0 +1,30 @@
+{{ if eq .Page.Params.show_jumbotron true }}
+
+<div class="container-fluid vertical-center custom-jumbotron">
```

Comment:
We can likely make use of the IoT jumbotron for this, and add the button tiles under the jumbotron like we do the featured-stats section on IoT

Reply:
I have separate the main menu to be shortcode `layouts/shortcodes/home-menu-dark.html`.
But for the jumbotron, I somehow think it would more similar to openhwgroup and openadx if we want to keep the jumbotron bg separate from navbar header. What do you think? 

I can definitely add alternative-bg later.

Reply:
I use IoT as my example as they have the text pulled to the right of center, rather than be centered. Adding or removing the white background to the nav is typically a single class with our current layout, typically set in the site configuration. They seem to intentionally pull it to keep the text from intersecting with the banner image content.

Reply:
IC, thx, let me check the iot styling and config.

Reply:
Fixed the header and jumbotron bg